src/csharp/Microsoft.Spark/ML/Feature/CountVectorizerModel.cs

```diff
@@ -0,0 +1,170 @@
+                s_countVectorizerModelClassName, uid, vocabulary))
+        {
+        }
+        
+        internal CountVectorizerModel(JvmObjectReference jvmObject) : base(jvmObject)
+        {
+        }
+
+        JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
+        
+        /// <summary>
+        /// Loads the <see cref="CountVectorizerModel"/> that was previously saved using Save
+        /// </summary>
+        /// <param name="path">
+        /// The path the previous <see cref="CountVectorizerModel"/> was saved to
+        /// </param>
+        /// <returns>New <see cref="CountVectorizerModel"/> object</returns>
+        public static CountVectorizerModel Load(string path) =>
+            WrapAsCountVectorizerModel((JvmObjectReference)
+                SparkEnvironment.JvmBridge.CallStaticJavaMethod(
+                    s_countVectorizerModelClassName,"load", path));
+        
+        /// <summary>
+        /// Gets the binary toggle to control the output vector values. If True, all nonzero counts
+        /// (after minTF filter applied) are set to 1. This is useful for discrete probabilistic
+        /// models that model binary events rather than integer counts. Default: false
+        /// </summary>
+        /// <returns>boolean</returns>
+        public bool GetBinary() => (bool)_jvmObject.Invoke("getBinary");
+
+        /// <summary>
+        /// Sets the binary toggle to control the output vector values. If True, all nonzero counts
+        /// (after minTF filter applied) are set to 1. This is useful for discrete probabilistic
+        /// models that model binary events rather than integer counts. Default: false
+        /// </summary>
+        /// <param name="value">Turn the binary toggle on or off</param>
+        /// <returns>
+        /// <see cref="CountVectorizerModel"/> with the new binary toggle value set
+        /// </returns>
+        public CountVectorizerModel SetBinary(bool value) =>
+            WrapAsCountVectorizerModel((JvmObjectReference)_jvmObject.Invoke("setBinary", value));
+
+        /// <summary>
+        /// Gets the column that the <see cref="CountVectorizerModel"/> should read from and
+        /// convert into buckets. This would have been set by SetInputCol
+        /// </summary>
+        /// <returns>string, the input column</returns>
+        public string GetInputCol() => _jvmObject.Invoke("getInputCol") as string;
+        
+        /// <summary>
+        /// Sets the column that the <see cref="CountVectorizerModel"/> should read from.
+        /// </summary>
+        /// <param name="value">The name of the column to as the source.</param>
+        /// <returns><see cref="CountVectorizerModel"/> with the input column set</returns>
+        public CountVectorizerModel SetInputCol(string value) =>
+            WrapAsCountVectorizerModel(
+                (JvmObjectReference)_jvmObject.Invoke("setInputCol", value));
```

Comment:
```suggestion
            WrapAsCountVectorizerModel((JvmObjectReference)_jvmObject.Invoke("setInputCol", value));
```

Reply:
Can I just check this one? If I do this it puts the semi-colon outside the line limit, maybe I need to set my ide to 101 though?